gemrb/core/Core.cpp

```diff
@@ -272,7 +272,7 @@ bool WithinPersonalRange(const Scriptable *actor, const Point &dest, int distanc
 int EARelation(const Scriptable* Owner, const Actor* target)
 {
 	ieDword eao = EA_ENEMY;
-	const Actor* actor = Owner->As<const Actor>();
+	const Actor* actor = Scriptable::As<Actor>(Owner);
```

Comment:
Should be `const Actor` still, so we won't get analyzer warnings from casting away the const.

Reply:
Oops.